lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -44,6 +44,12 @@
     <% mode_class = '' %>
   % endif
 <div class="course-container">
+ % if course_program_info:
+  <div class="label-xseries-association">
+    <i class="xseries-icon"></i>
+    <p class="message-copy">${_("XSeries Program Course")}</p>
```

Comment:
would it be possible to generate this string in the Python function as well, rather than hardcoding it here?